07.ls_object/lib/display.rb

```diff
@@ -74,9 +72,7 @@ def render_short_list
     file_count = @file_statuses.count
     row_count = (file_count / COLUMN_COUNT.to_f).ceil
     (COLUMN_COUNT - (file_count % COLUMN_COUNT)).times { resized_base_names << '' } if file_count % COLUMN_COUNT != 0
-    short_list_lines = []
-    resized_base_names.each_slice(row_count) { |file| short_list_lines << file }
-    short_list_lines.transpose.map { |line| line.join.rstrip }
+    resized_base_names.each_slice(row_count).to_a.transpose.map { |line| line.join.rstrip }.join("\n")
```

Comment:
```suggestion
    resized_base_names
      .each_slice(row_count)
      .to_a
      .transpose
      .map { |line| line.join.rstrip }
      .join("\n")
```

メソッドチェーンは横に長いと読みづらいので、たくさんチェーンする場合は`.`で改行するのがいいと思います。